aopy/preproc/wrappers.py

```diff
@@ -370,3 +384,49 @@ def proc_lfp(data_dir, files, result_dir, result_filename, overwrite=False, max_
                                                 max_memory_gb=max_memory_gb, **filter_kwargs)
         
     aodata.save_hdf(result_dir, result_filename, lfp_metadata, "/lfp_metadata", append=True)
+
+
+
+def proc_emg(data_dir, files, result_dir, result_filename, overwrite=False, max_memory_gb=1., filter_kwargs={}):
+    '''
+    Process emg data:
+        Loads 'broadband' hdf data or 'ecube' headstage data and metadata
+    Saves lfp data into the HDF datasets:
+        lfp_data (nt, nch)
+        lfp_metadata (dict)
+    
+    Args:
+        data_dir (str): where the data files are located
+        files (dict): dictionary of filenames indexed by system
+        result_filename (str): where to store the processed result
+        overwrite (bool, optional): whether to remove existing processed files if they exist
+        max_memory_gb (float, optional): max memory used to load binary data at one time
+        filter_kwargs (dict, optional): keyword arguments to pass to :func:`aopy.precondition.filter_lfp`
+    '''  
+    # Check if a processed file already exists
+    filepath = os.path.join(result_dir, result_filename)
+    if not overwrite and os.path.exists(filepath):
+        contents = aodata.get_hdf_dictionary(result_dir, result_filename)
+        if "emg" in contents:
```

Comment:
might need to double check this, i think it should be:
```suggestion
        if "emg_data" in contents:
```